api/tests/unit/infrastructure/serializers/jsonapi/training-serializer_test.js

```diff
@@ -39,7 +49,19 @@ describe('Unit | Serializer | JSONAPI | training-serializer', function () {
         thematics: [thematic1, thematicInAnotherCompetence],
         triggerTubes: [trainingTriggerTube1, anotherTrainingTriggerTube],
       });
-      const training = domainBuilder.buildTrainingForAdmin({ id: trainingId, trainingTriggers: [trainingTrigger] });
+      const trainingTrigger2 = domainBuilder.buildTrainingTriggerForAdmin({
+        id: trainingTriggerId2,
+        trainingId,
```

Comment:
💨 [dust] (suggestion/non-blocking) - pour etre plus lisable et pour se conformer au reste de les arguments, on pourrait utiliser named arguments pour tout
```suggestion
        trainingId: trainingId,
```